docs/src/refs.bib

```diff
@@ -37,3 +398,160 @@ @Book{OSCAR
   month         = {8},
   url           = {https://link.springer.com/book/9783031621260}
 }
+
+@Article{SF73,
+  author        = {Simpson, William A. and Frame, J. Sutherland},
+  title         = {The character tables for {SL}(3,q), {SU}(3,q{{\(^2\)}}), PSL(3,q), PSU(3,q{{\(^2\)}})},
+  zbl           = {0264.20010},
+  journal       = {Can. J. Math.},
+  fjournal      = {Canadian Journal of Mathematics},
+  volume        = {25},
+  pages         = {486--494},
+  year          = {1973},
+  doi           = {10.4153/CJM-1973-049-7},
+  language      = {English},
+  zbmath        = {3416061}
+}
+
+@Article{Sch07,
+  author        = {Schur, I.},
+  title         = {Untersuchungen {\"u}ber die {Darstellung} der endlichen {Gruppen} durch gebrochene lineare
+                  {Substitutionen}.},
+  journal       = {J. Reine Angew. Math.},
+  fjournal      = {Journal f{\"u}r die Reine und Angewandte Mathematik},
+  volume        = {132},
+  pages         = {85--137},
+  year          = {1907},
+  doi           = {10.1515/crll.1907.132.85},
+  url           = {https://eudml.org/doc/149256},
+  language      = {German},
+  zbmath        = {2643366},
+  jfm           = {38.0174.02}
+}
+
+@Article{Shi74,
+  author        = {Shinoda, Ken-ichi},
+  title         = {The conjugacy classes of {Chevalley} groups of type {{\((F_4)\)}} over finite fields of
+                  characteristic 2},
+  zbl           = {0306.20013},
+  journal       = {J. Fac. Sci., Univ. Tokyo, Sect. I A},
+  fjournal      = {Journal of the Faculty of Science. Section I A},
+  volume        = {21},
+  pages         = {133--159},
+  year          = {1974},
+  language      = {English},
+  zbmath        = {3479126}
```

Comment:
Check for DOI at MathSciNet

Reply:
Apparently also know doi:
```
@article {MR349863,
    AUTHOR = {Shinoda, Ken-ichi},
     TITLE = {The conjugacy classes of {C}hevalley groups of type
              {$(F\sb{4})$} over finite fields of characteristic {$2$}},
   JOURNAL = {J. Fac. Sci. Univ. Tokyo Sect. IA Math.},
  FJOURNAL = {Journal of the Faculty of Science. University of Tokyo.
              Section IA. Mathematics},
    VOLUME = {21},
      YEAR = {1974},
     PAGES = {133--159},
      ISSN = {0040-8980},
   MRCLASS = {20G40},
  MRNUMBER = {349863},
MRREVIEWER = {R.\ W.\ Carter},
}
```

Reply:
Indeed! Well it took me quite some effort but the DOI is 10.15083/00039769